### PR TITLE
Evohome HGI80 updates to add heat demand devices and All Sensors functions

### PR DESCRIPTION
**Pull request by Evsdd**
Hi,

I've completed testing on a few updates to the Evohome HGI80 usb device functionality.  I'm not sure how to handle the one update made to the compressed domoticz.js.gz file which allows the Evohome 'Zone Relay' switch devices to display the %level values in the Switch log.  Let me know if you need the uncompressed version.

Dan

### CODE REVIEW

**Comment by gizmocuz** [comment]
Hi Dan,

Yes, we should somehow use the uncompressed version of this file, as more and more people are changing it.
I will try to work on this.
If you write down your changes, could you make the patch again with the latest domoticz.js.gz from the repository ? Else write your changes to this file?

File: msbuild/domoticz.sln [hunk 1]
```diff
@@ -11,8 +11,8 @@ Global
 		Release|Win32 = Release|Win32
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{6AD96441-5B2C-467B-AAB6-A1B8368D503A}.Debug|Win32.ActiveCfg = Debug|Win32
-		{6AD96441-5B2C-467B-AAB6-A1B8368D503A}.Debug|Win32.Build.0 = Debug|Win32
+		{6AD96441-5B2C-467B-AAB6-A1B8368D503A}.Debug|Win32.ActiveCfg = Release|Win32
```
Comment (gizmocuz) [inline msbuild/domoticz.sln]: I dont think this change is valid... No more debug ...

**Comment by Evsdd** [comment]
Sorry about that.  I've corrected the Domoticz.sln file to reinstate the debug configuration and reapplied my changes to the most recent version of Domoticz.js.gz

**Comment by gizmocuz** [comment]
Thanks!

**Pull request merged**